releasenotes/notes/prepare-1.4.3-5595d66661012aaf.yaml

```
@@ -0,0 +1,3 @@
+---
+prelude: >
+    Qiskit 1.4.3 is a small patch release, fixing some bugs found in the 1.4 extended support series.
```

What constitutes a small release?

---

Channeling my inner @beckykd and @abbycross don't we prefix versions with `v` in the documentation now?

```suggestion
    Qiskit v1.4.3 is a small patch release, fixing some bugs found in the v1.4.x release series.
```

---

> What constitutes a small release?

Copy-paste from a previous small release prelude :)